apps/studio/schemas/boardYear.ts

```diff
@@ -0,0 +1,67 @@
+import { HeartIcon } from "@sanity/icons";
+
+const organizerSearch = [
+  {
+    type: "object",
+    fields: [
+      {
+        name: "organizer",
+        type: "reference",
+        title: "Organizer",
+        to: {
+          type: "organizer",
+        },
+        validation: (Rule) => Rule.required(),
+      },
+      {
+        name: "position",
+        type: "string",
+        title: "Position",
+        validation: (Rule) => Rule.required(),
+      },
+    ],
+    preview: {
+      select: {
+        title: "organizer.name",
+        subtitle: "position",
+      },
+    },
+    icon: HeartIcon,
+  },
+];
+
+const nameTitleAssoc: Map<string, string> = new Map([
+  ["corporate", "Corporate Organizers"],
+  ["logistics", "Logistics Organizers"],
+  ["marketing", "Marketing Organizers"],
+  ["tech", "Tech Organizers"],
+]);
+
+export default {
+  name: "boardYears",
```

Comment:
This should probably be `boardYear`.

Reply:
Thanks for catching this! Fixed in [`babb28d`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/babb28de063249dfcad0d491a4467125cb60e11d)